Dockerfile

```diff
@@ -1,18 +1,38 @@
-FROM quay.io/3scale/ruby:2.1
-MAINTAINER Oriol Martí <oriol@3scale.net> # 2015-06-26
+FROM centos:7 
+LABEL authors="Oriol Martí <oriol@3scale.net>,Daniel Cesario <dcesario@redhat.com>"
 
-RUN apt-install -y libssl-dev
+ENV RUBY_VERSION="rh-ruby23"
+
+RUN yum -y update \
+	&& yum install -y centos-release-scl \
+	&& yum install -y ${RUBY_VERSION} ${RUBY_VERSION}-ruby-devel \
```

Comment:
For Some weird reason, the `yum install -y centos-release-scl` dont works well if you install the other packets on the same command. I will isolate this command and install the other packages in the same instruction.